portfolio/src/main/webapp/index.html

```diff
@@ -30,10 +33,8 @@ <h2>Help me <button onclick="grow()">Grow</button></h2>
     <p id="endMsg"></p>
     <br>
     <h2>Comments</h2>
-    <form>
-      <label>Leave a comment </label>
-      <input type="text" id="new-comment" name="new-comment">
-      <button type="button" onclick="submitComment()">Submit</button>
+    <form id="leave-a-comment">
```

Comment:
id should be a noun and not a verb. Maybe something like `comment-submission-form`. Change variables related to this in JS as well.

Reply:
👌 will stick to the style guide